### ISSUE TITLE
feature request: deleting bins

### ISSUE THREAD

**Issue by BixelPitch** [zuplo/mockbin#37]
Hello there!

## TL;DR:
Currently it is only possible to create bins - Deleting a bin is not possible (yet). I had to work on a problem, where I had to alter the response of a bin slightly each try - So I ended up with creating lots and lots of new bins every time. So my list of recent bins got really long. Would have been handy to have the possebility of directly deleting my old bins from the list - even if they would be still there, but hidden from my eye. I know I can clear my browser data, but then all of the entries are gone. Additionally a optional TTL would be nice when creating a new bin. 

## Request
- Make entries in the "recent bins" list deletable.
- Add an option to set a TTL before creating a bin. So the user can be sure the bin will automatically delete itself in x days.

**Comment by AdrianMachado** [comment]
Apologies for the late reply here! These are great suggestions @BixelPitch - we are doing some revamp around Mockbin so I will try and include this
